Tests/ContentsquareInstanceTests.swift

```diff
@@ -15,69 +15,311 @@ import XCTest
 
 class ContentsquareInstanceTests: XCTestCase {
 
-    let contentsquareInstance = MockContentsquareInstance()
+    var contentsquareInstance: MockContentsquareInstance!
```

Comment:
This is not needed. The entire ContentsquareInstanceTest class is recreated at every test. Can be reverted.